scripts/gen_chapters.js

```diff
@@ -0,0 +1,52 @@
+/* jshint node:true */
+"use strict";
+
+var _ = require('lodash'), 
+  path = require('path'),
+  fs = require('fs'),
+  map_config_path = process.argv[4] ? process.argv[4] :
+  path.join(__dirname, 'map_config.json'),
+  map_config = require(map_config_path),
+  dat = fs.readFileSync(map_config_path, 'utf8'),
+  map_data = {},
+  map_id = process.argv[2],
+  width = map_config.width,
+  tile_width = map_config.tileWidth,
+  tile_height = map_config.tileHeight,
+  map_length = map_config.maps[0].length,
+  map_repeat = map_config.maps[0].repeat,
+  bridge_length = map_config.bridge.length,
+  last_chapter_height = map_config.height,
+  first_chapter_height = map_length * map_repeat + bridge_length,
+  total_chapters = map_config.length,
+  data = fs.readFileSync('maps/final_map.js', 'utf8'),
+  chapters = [],
+  id = 1,
+  grid;
+
+for (var i = map_config.height; i >= first_chapter_height; i-=first_chapter_height) {
+  chapters.push(i);
+}
+chapters = _.map(chapters, function (val) {return val - 3;});
```

Comment:
why can't we do this in the previous for loop itself ?
Also add a comment on why you are subtracting by 3.

Will be difficult to understand in the future why there is a magic number 3